terminate called after throwing an instance of 'Util::CompilerBug'
  what():  COMPILER BUG: ../frontends/p4/typeChecking/typeChecker.h:100
../testdata/p4_16_errors/header3.p4(18): Field tuple<bit<1>, bit<1>> field1 of header H cannot have type tuple<bit<1>, bit<1>>
    tuple<bit, bit> field1;
                    ^^^^^^
../testdata/p4_16_errors/header3.p4(18)
    tuple<bit, bit> field1;
    ^^^^^^^^^^^^^^^

